code/game/gamemodes/miniantags/sintouched/objectives.dm

```diff
@@ -32,4 +32,18 @@
 	explanation_text = "You are the BEST thing on the station.  Make sure everyone knows it."
 
 /datum/objective/sintouched/acedia
-	explanation_text = "Angels, devils, good, evil... who cares?  Just ignore any hellish threats and do your job."
\ No newline at end of file
+	explanation_text = "Angels, devils, good, evil... who cares?  Just ignore any hellish threats and do your job."
+
+/datum/objective/sintouched/engine
+	explanation_text = "Go have a good conversation with the Singularity/Tesla/Supermatter crystal. Bonus points if it responds."
+	
+/datum/objective/sintouched/teamredisbetterthangreen
+	explanation_text = "Tell the AI (or a borg/pAI/drone if there is no AI) some corny technology jokes until it cries for help."
+	
+/datum/objective/sintouched/time
+	if(istype(ticker.mode, /datum/game_mode/clock_cult))
```

Comment:
the error is here, you're trying to run code in a definition, you can't do that, so it sees ```/datum/game_mode/clock_cult``` as a new definition, which also isn't allowed as clock cult is already defined.

A simple fix would be to add ```/New()``` to the end of line 43 so it becomes ```/datum/objective/sintouched/time/New()```, and then on the line just below it (moving everything else down a line) adding ```..()```